script/docker-murmur

```
@@ -6,7 +6,7 @@ ICEFILE="/etc/murmur/ice.ini"
 WELCOMEFILE="/data/welcome.txt"
 
 setVal() {
-    if [ ${1} ] && [ ${2} ]; then
+    if [ ${1} ] && [ "${2}" ]; then
```

Actually, I'd prefer leaving the double quotes off of the test for the first argument. The first argument in this case is the option name -- if someone modifies the script locally and attempts to pass in an option name with a space in it, well, I think they _should_ experience an error instead of silently failing to set the value.